dist/settings.json

```
@@ -378,6 +378,12 @@
       "CommandName": "ToggleMute",
       "Commands": [
         "mute"
+      ],
```

```suggestion
      ]
```
Remove trailing comma